src/main/java/seedu/address/ui/StartupCard.java

```diff
@@ -72,6 +79,9 @@ public StartupCard(Startup startup, int displayedIndex) {
         industryAndFundingStage.getChildren().addAll(
                 new Label(startup.getIndustry().value),
                 new Label(fundingLevel));
+        valuationDisplay.getChildren().addAll(
+                new Label("Valuation: " + Valuation.reformatValuation(startup.getValuation().value))
+        );
```

Comment:
Maybe you can do something like
```suggestion
        valuation.setText("Valuation: " + Valuation.reformatValuation(startup.getValuation().value));
```